PopulateTools/gobierto
# Remove filtering text option

Closes #1339

### What does this PR do?

This PR adds a new option to filtering rules to remove the text used to filter in the processed event.

It's not finished since I need some help with the CSS @furilo @Crashillo:

![screen shot 2018-01-18 at 15 06 06](https://user-images.githubusercontent.com/17616/35101814-296647dc-fc61-11e7-9474-ec75ccb84bf4.png)

### How should this be manually tested?

1. Create a rule
2. Activate the new option
3. Check the text is removed from the title or the description
